ui-library-starting-with-calling-stateful/package.json

```diff
@@ -3,7 +3,9 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@azure/communication-react": "^1.0.0-beta.7",
+    "@azure/communication-calling": "1.3.2",
+    "@azure/communication-chat": "1.1.0",
+    "@azure/communication-react": "latest",
```

Comment:
what happens when latest for communication-react only supports some later stable/beta version?

should we stay on latest  for communication-react in this sample when we have peer dependencies not moving as fast?

Reply:
Right you are. Let's pin to `1.0.0` for now. I was simply making them consistent.